content-src/components/TopSites/SearchShortcutsForm.jsx

```diff
@@ -38,7 +38,10 @@ export class SearchShortcutsForm extends React.PureComponent {
     const shortcuts = [];
     const {rows, searchShortcuts} = props.TopSites;
     searchShortcuts.forEach(shortcut => {
-      shortcuts.push(Object.assign({}, shortcut, {isSelected: !!rows.find(({isPinned, searchTopSite, label}) => isPinned && searchTopSite && label === shortcut.keyword)}));
+      shortcuts.push({
```

Comment:
Found this while I was testing

Reply:
Basically if there is `null` because of a pin space, this throws

Reply:
Yay! Someday I’ll get all that new syntax into my muscle memory.

Reply:
Yep. the empty slots bite again.

Reply:
Do we also need to fix this in onSaveButtonClick?

Reply:
yes we do